modules/search/instant-search/components/search-app.jsx

```diff
@@ -195,7 +219,7 @@ class SearchApp extends Component {
 							locale={ this.props.options.locale }
 							query={ getSearchQuery() }
 							response={ this.state.response }
-							resultFormat={ this.props.options.resultFormat }
+							resultFormat={ resultFormat }
```

Comment:
```suggestion
							resultFormat={ getResultFormatQuery() }
```

Nit: No need to instantiate a new const here.